spec/webview-spec.js

```diff
@@ -1068,7 +1068,7 @@ describe('<webview> tag', function () {
       await loadWebView(webview, { src })
 
       const data = await webview.printToPDF({})
-      expect(data).to.be.an.instanceof(Buffer).that.is.not.empty()
+      expect(data).to.be.an.instanceof(Uint8Array).that.is.not.empty()
```

Comment:
the documentation for `<webview>.printToPDF(options)` should specify the return value as `Promise<Uint8Array >` instead of `Promise<Buffer>`

Reply:
Unfortunately this currently produces an error in our typings generator:

```
../electron.d.ts(10212,62): error TS2694: Namespace 'Electron' has no exported member 'Uint8Array'.
```

Reply:
https://github.com/electron/typescript-definitions/pull/156 fixes

Reply:
electron/typescript-definitions#157 _actually_ fixes.